writeengine/server/we_readthread.cpp

```diff
@@ -156,11 +156,13 @@ void DmlReadThread::operator()()
           break;
         }
 
+#if 0
```

Comment:
This should be a remnant of bulk insertion API.

Reply:
I did not find any references to `WE_SVR_BATCH_INSERT_BINARY` in the engine code, so I commented this out. Is the bulk insertion API or any external library still dependent on this code? If not, I can remove this.

Reply:
Same thing. This is used by the bulk insertion API.

Reply:
Done. I have uncommented back this change.